index.html

```diff
@@ -0,0 +1,612 @@
+                <li>
+                  <a>Education</a>
+                </li>
+                <li>
+                  <a>Food</a>
+                </li>
+                <li>
+                  <a>Health</a>
+                </li>
+                <li>
+                  <a>Jobs</a>
+                </li>
+                <li>
+                  <a>Magazine</a>
+                </li>
+                <li>
+                  <a>Parenting</a>
+                </li>
+                <li>
+                  <a>Real Estate</a>
+                </li>
+                <li>
+                  <a>Style</a>
+                </li>
+                <li>
+                  <a>T Magazine</a>
+                </li>
+                <li>
+                  <a>Travel</a>
+                </li>
+                <li>
+                  <a>Love</a>
+                </li>
+              </ul>
+            </div>
+            <div class="floating-links">
+              <ul class="column-5">
+                <li><strong>MORE</strong></li>
+                <li>
+                  <a>Reader Center</a>
+                </li>
+                <li>
+                  <a>Wirecutter</a>
+                </li>
+                <li>
+                  <a>Live Events</a>
+                </li>
+                <li>
+                  <a>The Learning Network</a>
+                </li>
+                <li>
+                  <a>Tools &amp; Services</a>
+                </li>
+                <li>
+                  <a>N.Y.C. Events Guide</a>
+                </li>
+                <li>
+                  <a>Multimedia</a>
+                </li>
+                <li>
+                  <a>Photography</a>
+                </li>
+                <li>
+                  <a>Video</a>
+                </li>
+                <li>
+                  <a>Newsletters</a>
+                </li>
+                <li>
+                  <a>NYT Store</a>
+                </li>
+                <li>
+                  <a>Times Journeys</a>
+                </li>
+                <li>
+                  <a>Manage My Account</a>
+                </li>
+              </ul>
+            </div>
+            <hr class="vertical-column-line">
+            <div class="floating-links">
+              <div class="column-6">
+                
+                <h3><strong>SUBSCRIBE</strong></h3>
+                   <ul class="sub-1">
+                      <li>
+                        <a class="bottom-links"><svg class="bottom-links-image" viewBox="0 0 14 13" fill="#000"><path d="M13.1,11.7H3.5V1.2h9.6V11.7zM13.1,0.4H3.5C3,0.4,2.6,0.8,2.6,1.2v2.2H0.9C0.4,3.4,0,3.8,0,4.3v5.2v1.5c0,0.8,0.8,1.5,1.8,1.5h1.7h0h7.4h2.2c0.5,0,0.9-0.4,0.9-0.9V1.2C14,0.8,13.6,0.4,13.1,0.4"></path><polygon points="10.9,3 5.2,3 5.2,3.9 11.4,3.9 11.4,3"></polygon><rect x="5.2" y="4.7" width="6.1" height="0.9"></rect><rect x="5.2" y="6.5" width="6.1" height="0.9"></rect></svg> <b class="bottom-links-headings">Home Delivery</b></a>
```

Comment:
- [ ]  put height and width selectors in a class or Id in your CSS file as no inline styling is permitted in the HTML